content/flare/consensus.md

```diff
@@ -287,3 +287,223 @@ Market forces cause quorum slices to overlap and this gives rise to the network-
+The set of transactions that each one sees is a growing blockchain that does not have a built-in notion of finality and the streams of data are an advancing sequence of blockchain "tips" that each of the participants sees.
+
+The problem of ordering transactions in asynchronous types of protocols, where they are received with some time delay and in mixed order, is also known as _Atomic Broadcast_.
+It describes the situation where a set of nodes commonly constructs a total ordering of a set of transactions, where these transactions might not arrive all at the same time.
+Every time a node receives a transactions, it processes it and places it at its position.
+The sequence of transactions must be output in an incremental manner, meaning that before putting a transaction at some position, previous positions must have their respective transactions set.
+It is assumed that whenever an honest node outputs a transaction at some position, every other honest node will also eventually output the same transaction for this position.
+Also, every transaction that is input at some honest node is eventually output by all honest nodes.
+This ensures that not only all honest nodes will produce the same ordering, but also that no transactions is lost due to censorship because all honest nodes will output it.
+
+So Aleph produces a single stream of data that "combines" all the individual streams of the nodes and is consistent among them, meaning the nodes produce the same unique sequence of finalized blocks.
+As a Byzantine Tolerant type of protocol, Aleph implements Atomic Broadcast over a set of `3f+1` number of nodes, where `f` denotes the number of dishonest nodes.
+The protocol deals with an expected latency for each transaction, which is the time it is output by all honest nodes after it has been input to at least some preset number of nodes.
+
+The consensus separates the network communication from the protocol logic.
+The only role for the network layer is to maintain a common data structure in the form of a DAG tree among all nodes.
+The purpose of using it is to divide the algorithm execution into separate virtual rounds.
+In each round the nodes emits exactly one "unit" that should be thought of as a message broadcast to all the other nodes, and it is a vertex position on its local DAG version.
+Every unit contains information about its creator, its parents and additional data that can be included in it.
+All the nodes are expected to generate such units and maintain their local copies of the common DAG, to which new units are continuously added.
+Aleph uses transitive properties of the underlying DAG.
+Each unit has a "DAG-round" that is defined by the maximum length of a downward chain starting from the current transaction, meaning getting as long as possible path of its ancestors.
+A unit with no parents will have a DAG-round 0, otherwise it has a DAG-round equal to the maximum of DAG-rounds of its parents plus one.
```

Comment:
```suggestion
A unit with no parents has a DAG-round of `0`, otherwise it has a DAG-round equal to the maximum of DAG-rounds of its parents plus `1`.
```

Reply:
Fixed here https://github.com/optakt/optakt.github.io/pull/25/commits/e51539edef429ee0583f05806acf8610dd0971a0#diff-af4a1f964639da79798e8b30e7b99b5fb3f29ea0d4da47acd88f0d9c9867be13R414
Thanks!